### DISCUSSION TITLE
Configuration of "Adguard parental control"

### DISCUSSION THREAD

**Discussion by jkoopmann** [AdguardTeam/AdGuardHome#4914] [General]
Hi,

I might be missing something extremely obvious. However in my freshly installed adguard home installation I am able to enable/disable "Adguard parental control" however I am not able to configure anything else. All configuration screens I was able to find in the net seem to be about the other Adguard products.

Any help is greatly appreciated.

Regards
   JP

**Comment by ainar-g** [comment]
How did you install AdGuard Home?  What screens do you see?

**Reply by jkoopmann** [reply]
FreeBSD Port. I do not see any parental screens other than the option to enable it in the general section.

**Reply by ainar-g** [reply]
That is the only option.  It is either on or off.  What exactly would you like to configure?

**Reply by jkoopmann** [reply]
I was expecting settings like

https://adguard.com/en/article/parental-control-software.html

e.g. sensitivity.


Regards
   JP

**Reply by ainar-g** [reply]
Ah, I see.  I'm not sure if this is a setting that we could support, but if you really need it, you can file a feature request / enhancement issue, and we'll see if there is support for adding these settings.
